requirements/development.in

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+-r base.txt
+-e file:.[cors,druid,hive,mysql,postgres,thumbnails]
```

Comment:
Should we add presto here?

Reply:
debatable, because we are already using local dev docker based on development.txt and it already has lot's of "unnecessary" dependencies there